FeatureTest.java

```diff
@@ -6,7 +6,8 @@ public static void main(String[] args) {
     public int square(int variable){
         String str = null;
         System.out.println(str.length());
-        str = "fjff";
+        String newer = null;
+        System.out.println(newer.length());
         return variable * variable;
```

Comment:
[code-reviewer[bot]]
Null pointer dereference: Variable 'newer' is null (assigned null at line 9)